projects/ng-table-virtual-scroll/src/lib/table-item-size.directive.ts

```diff
@@ -86,7 +86,7 @@ export class TableItemSizeDirective implements OnChanges, AfterContentInit, OnDe
       .reduce((prevState, state) => prevState && state, true);
   }
 
-  ngAfterContentInit() {
+  ngAfterViewInit() {
```

Comment:
had some troubles with the header calculation with the AfterContentInit lifecycle hook, worked better with this one